Library/High Tech/High Tech Equipment.eqp

```diff
@@ -37720,6 +37741,153 @@
 				"extended_weight": "4.6 lb"
 			}
 		},
+		{
+			"id": "eXZa2m9BxAi52geH_",
+			"description": "Pilot's Survival Vest",
+			"reference": "HT59",
+			"local_notes": "Jungle survival kit.",
+			"tech_level": "7",
+			"legality_class": "4",
+			"tags": [
+				"General Equipment"
+			],
+			"base_value": "250",
+			"base_weight": "5",
+			"quantity": 1,
+			"equipped": true,
+			"calc": {
+				"value": 250,
+				"extended_value": 250,
+				"weight": "5 lb",
+				"extended_weight": "5 lb"
+			}
+		},
+		{
+			"id": "eBGjhGjn8TxacJohU",
+			"description": "Personal Basics",
+			"reference": "HT58",
+			"local_notes": "Status 0; @Environment@ Survival specialty.",
+			"tech_level": "5",
+			"legality_class": "4",
+			"tags": [
+				"General Equipment"
+			],
+			"base_value": "6",
+			"base_weight": "1",
+			"quantity": 1,
+			"equipped": true,
+			"calc": {
+				"value": 6,
+				"extended_value": 6,
+				"weight": "1 lb",
+				"extended_weight": "1 lb"
+			}
+		},
+		{
+			"id": "eJ5Vp6njReziLM4X9",
+			"description": "Personal Basics",
+			"reference": "HT58",
+			"local_notes": "Status 1; @Environment@ Survival specialty.",
```

Comment:
Interesting that you split them up by Status since Status affects Cost of Living.  I probably would have just used the Status 0 cost of living to price Personal Basics.  I think it's fine though; if a GM wants to require that high-status people buy more expensive Personal Basics they can, and if they don't want to they can just let everyone use the Status 0 version.